src/clients/ftp-server.rb

```diff
@@ -443,20 +387,19 @@ def main
       Builtins.y2milestone("[ftp-server] module finished")
       Builtins.y2milestone("----------------------------------------")
 
-      deep_copy(@ret) 
+      deep_copy(@ret)
 
       # EOF
     end
 
-    def FTPdCMDShow(options)
-      options = deep_copy(options)
+    def FTPdCMDShow(_options)
       CommandLine.Print("")
       CommandLine.Print(String.UnderlinedHeader(_("Display Settings:"), 0))
       CommandLine.Print("")
-      #start-up settings
+      # start-up settings
       CommandLine.PrintNoCR(_("Start-Up:"))
       if GetStartedViaXinetd()
```

Comment:
Shouldn't this call be removed as commented below with Set method?

Reply:
why it should be removed? When systemd socket start ftp daemon, then it should be written. There are three states: socket, service and manual and this is one of the states.

Reply:
As with Set I will rename it to correspond the behavior with the naming